src/Middleware/Middleware.slnf

```diff
@@ -32,6 +32,7 @@
       "src\\Middleware\\HostFiltering\\src\\Microsoft.AspNetCore.HostFiltering.csproj",
       "src\\Middleware\\HostFiltering\\test\\Microsoft.AspNetCore.HostFiltering.Tests.csproj",
       "src\\Middleware\\HttpLogging\\samples\\HttpLogging.Sample\\HttpLogging.Sample.csproj",
+      "src\\Middleware\\HttpLogging\\samples\\Logging.W3C.Sample\\Logging.W3C.Sample.csproj",
```

Comment:
For some reason this project is showing up at the top level when opening the slnf

Reply:
I also see that. The following should fix it:

```diff
diff --git a/AspNetCore.sln b/AspNetCore.sln
index 08261d8216..f1e9736e42 100644
--- a/AspNetCore.sln
+++ b/AspNetCore.sln
@@ -8564,6 +8564,7 @@ Global
                {558C46DE-DE16-41D5-8DB7-D6D748E32977} = {3EC71A0E-6515-4A5A-B759-F0BCF1BCFC56}
                {B1AA24A4-5E02-4DC1-B57F-6EB03F91E4DD} = {44963D50-8B58-44E6-918D-788BCB406695}
                {5C641396-7E92-4F5C-A5A1-B4CDF480539B} = {088C37A5-30D2-40FB-B031-D163CFBED006}
+               {17459B97-1AA3-4154-83D3-C6BDC9FA3F85} = {022B4B80-E813-4256-8034-11A68146F4EF}
        EndGlobalSection
        GlobalSection(ExtensibilityGlobals) = postSolution
                SolutionGuid = {3E8720B3-DBDD-498C-B383-2CC32A054E8F}
```